docs/topics/lombok.md

```diff
@@ -0,0 +1,164 @@
+
+## Gradle
+
+Apply the `kotlin-plugin-lombok` Gradle plugin in the `build.gradle(.kts)` file:
+
+<tabs>
+
+```groovy
+plugins {
+    id 'org.jetbrains.kotlin.plugin.lombok' version '%kotlinVersion%'
+    id 'io.freefair.lombok' version '5.3.0'
+}
+```
+
+```kotlin
+plugins {
+    id ("org.jetbrains.kotlin.plugin.lombok") version "%kotlinVersion%"
+    id ("io.freefair.lombok") version "5.3.0"
+}
+```
+
+</tabs>
+
+See the [test project with examples of Lombok compiler plugin usage](https://github.com/kotlin-hands-on/kotlin-lombok-examples/tree/master/kotlin_lombok_gradle/nokapt).
+
+### Using the Lombok configuration file
+
+If you use [Lombok configuration file](https://projectlombok.org/features/configuration) `lombok.config`, 
+provide a path to it to the plugin. The path should be relative to the module's directory. 
+Add the following code to your `build.gradle(.kts)` file:
+
+<tabs>
+
+```groovy
+kotlinLombok {
+    lombokConfigurationFile file("lombok.config")
+}
+```
+
+```kotlin
+kotlinLombok {
+    lombokConfigurationFile(file("lombok.config"))
+}
+```
+
+</tabs>
+
+See the [test project with examples of Lombok compiler plugin and `lombok.config` usages](https://github.com/kotlin-hands-on/kotlin-lombok-examples/tree/master/kotlin_lombok_gradle/withconfig).
+
+## Maven
+
+To use Lombok compiler plugin, add the plugin `lombok` to the `compilerPlugins` section and the dependency 
+`kotlin-maven-lombok` to the `dependencies` section. 
+If you use [Lombok configuration file](https://projectlombok.org/features/configuration) `lombok.config`,
+provide a path to it to the plugin in the `pluginOptions`. Add the following lines to the `pom.xml` file:
+
+```xml
+<plugin>
+    <groupId>org.jetbrains.kotlin</groupId>
+    <artifactId>kotlin-maven-plugin</artifactId>
+    <version>${kotlin.version}</version>
+    <configuration>
+        <compilerPlugins>
+            <plugin>lombok</plugin>
+        </compilerPlugins>
+        <pluginOptions>
+            <option>lombok:config=${project.basedir}/lombok.config</option>
+        </pluginOptions>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-maven-lombok</artifactId>
+            <version>${kotlin.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.20</version>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+</plugin>
+```
+
+See the [test project example of Lombok compiler plugin and `lombok.config` usages](https://github.com/kotlin-hands-on/kotlin-lombok-examples/tree/master/kotlin_lombok_maven/nokapt).
+
+## Using the plugin along with kapt
+
+By default, [kapt](kapt.md) compiler plugin launches all annotation processors and disables javac's annotation processors.
+You need javac's annotation processors working to launch Lombok.
```

Comment:
For me it sounds weird. Maybe something like 'To run lombok along with kapt you need to tell it to keep javac's annotation processors'

Reply:
Does it work?
>To run [Lombok](https://projectlombok.org/) along with kapt, you need to set up kapt to keep javac's annotation processors working.